src/main/resources/metafile-schema.json

```diff
@@ -23,25 +23,25 @@
       "id": "#/properties/delivery_date",
       "type": "string",
       "title": "Date time in UTC for when the package was delivered at the bulk scanning supplier",
-      "pattern": "^[012][0-9]-[012][0-9]-[2][0][0-9][0-9] [012][0-9]:[0-5][0-9]:[0-5][0-9].[0-9]{6}$"
+      "pattern": "^([012][0-9]|30|31)-([0][0-9]|[1][012])-[2][0][0-9][0-9] ([01][0-9]|[2][0123]):[0-5][0-9]:[0-5][0-9].[0-9]{6}$"
     },
     "opening_date": {
       "id": "#/properties/opening_date",
       "type": "string",
```

Comment:
is it possible to have this as type date? and just require a ISO-8601 date string that java will accept?
this seems over complicated